docs/java-rest/high-level/security/authenticate.asciidoc

```diff
@@ -0,0 +1,56 @@
+[[java-rest-high-security-authenticate]]
```

Comment:
There is a new style of these #34203 should point you in the right direction.

Reply:
Also see #34187, which includes an example security doc that uses this new style.

Reply:
ACK